packages/opentelemetry-core/src/platform/browser/log-level.ts

```diff
@@ -0,0 +1,25 @@
+/**
+ * Copyright 2020, OpenTelemetry Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import { LogLevel } from '../../common/types';
+
+/**
+ * @returns the default log level for opentelemetry.
+ * TODO: https://github.com/open-telemetry/opentelemetry-js/issues/1096
+ */
+export function getLogLevel(): LogLevel {
```

Comment:
I think it would make more sense to call this function `getDefaultLogLevel()`

Reply:
Yeah, both these names are slightly misleading since it gets the default level or the (env/global*) var set by user log level.  With that said you still prefer   `getDefaultLogLevel()` as the least bad of the two?            

* global is TODO.. so effectively it is just returning the default log level today in the case of browser

Reply:
I would prefer `getDefaultLogLevel` indeed. It implies that it can be overriden for each component if needed